packages/Main/src/Layer/LayerUpdateStrategy.js

```diff
@@ -74,7 +69,7 @@ export function chooseNextLevelToFetch(strategy, node, nodeLevel = node.level, c
             // default strategy
             case STRATEGY_MIN_NETWORK_TRAFFIC:
             default:
-                nextLevelToFetch = _minimizeNetworkTraffic(node, nodeLevel, currentLevel, layer.source);
+                nextLevelToFetch = _minimizeNetworkTraffic(node, nodeLevel);
```

Comment:
If you remove `_minimizeNetworkTraffic`

```suggestion
     nextLevelToFetch = nodeLevel
```